lib/ex.coffee

```diff
@@ -161,7 +176,7 @@ class Ex
   wq: (args...) =>
     @write(args...).then => @quit()
 
-  x: (args...) => @wq(args...)
+  xit: (args...) => @wq(args...)
```

Comment:
Again, I mainly wanted to add specs, but I can add `:exit` along with a few more commands (`:wall`, `:qall`, `:xall`, `:update`, `:new` and `:vnew`). I'll also fix the issues raised in #80, #81 and #82, it's all already in atom/vim-mode#762